inf-clojure.el

```diff
@@ -682,11 +682,24 @@ to continue it."
 (defun inf-clojure-preoutput-filter (str)
   "Preprocess the output STR from interactive commands."
   (inf-clojure--log-string str "<-RES----")
-  (cond
-   ((string-prefix-p "inf-clojure-" (symbol-name (or this-command last-command)))
-    ;; Remove subprompts and prepend a newline to the output string
-    (inf-clojure-chomp (concat "\n" (inf-clojure-remove-subprompts str))))
-   (t str)))
+  (let* ((last-line (car (last (split-string str "\n"))))
```

Comment:
I guess you can also add some high-level description of the need for this pre-processing in the docstring or above the `let*`.